app/policies/attachment_policy.rb

```diff
@@ -0,0 +1,5 @@
+class AttachmentPolicy < ApplicationPolicy
+  def show?
+    user.manage_assessments?(record.attachable.outcome_coverage.coverage.course.department)
```

Comment:
We should set up a chain of delgations so you can do something like: `record.attachable.department`

This gets a little cmplicated because coverages themselves can have two different departments: The department of the course that added the coverage or the department of the subject. Those don't have to be the same thing. I'm dealing with this a bit in the work I'm doing for results.

I think it might make sense to do: `Coverage#course_department` and `Coverage#subject_department` so we are explicit about these things.

Then you'd have to set up methods/delegations so you could do `record.attachable.course_department`. Does this make sense?